Loop/Managers/DoseMath.swift

```diff
@@ -256,7 +256,8 @@ extension Collection where Element: GlucoseValue {
             // Compute the dose required to bring this prediction to target:
             // dose = (Glucose Δ) / (% effect × sensitivity)
 
-            let percentEffected = 1 - model.percentEffectRemaining(at: time)
+            // For 0 <= time <= effectDelay, assume a small amount effected. This will result in large unit recommendation rather than no recommendation at all.
+            let percentEffected = Swift.max(.ulpOfOne, 1 - model.percentEffectRemaining(at: time))
```

Comment:
Is there an associated unit test for this change?

Reply:
The tests in DoseMath did change for this, and cover this code.